go: error loading module requirements

63fc57b4ed95fc1181345b3fffa102f76ae43448 cannot be compiled with Go 1.12.4

```console
$ make ipcs
go: finding github.com/libp2p/go-libp2p-crypto v2.0.1+incompatible
go: finding github.com/libp2p/go-flow-metrics v0.2.0
go: finding github.com/libp2p/go-libp2p-protocol v1.0.0
go: finding github.com/libp2p/go-libp2p-metrics v2.1.7+incompatible
go: finding github.com/multiformats/go-multiaddr-net v1.6.3
go: finding github.com/libp2p/go-libp2p-peer v2.4.0+incompatible
go: finding github.com/multiformats/go-multihash v1.0.8
go: finding github.com/ipfs/go-ipfs-delay v0.0.0-20181109222059-70721b86a9a8
go: finding github.com/Kubuxu/go-os-helper v0.0.1
go: finding github.com/AndreasBriese/bbloom v0.0.0-20180913140656-343706a395b7
go: github.com/libp2p/go-flow-metrics@v0.2.0: unknown revision v0.2.0
go: finding github.com/go-check/check v0.0.0-20180628173108-788fd7840127
go: github.com/libp2p/go-libp2p-protocol@v1.0.0: unknown revision v1.0.0
go: github.com/multiformats/go-multiaddr-net@v1.6.3: unknown revision v1.6.3
go: finding github.com/whyrusleeping/chunker v0.0.0-20181014151217-fe64bd25879f
go: finding github.com/gxed/go-shellwords v1.0.3
go: github.com/libp2p/go-libp2p-crypto@v2.0.1+incompatible: unknown revision v2.0.1
go: finding github.com/dgraph-io/badger v1.5.5-0.20190226225317-8115aed38f8f
go: github.com/multiformats/go-multihash@v1.0.8: unknown revision v1.0.8
go: github.com/libp2p/go-libp2p-metrics@v2.1.7+incompatible: unknown revision v2.1.7
go: finding github.com/ipfs/go-ipfs-blockstore v0.0.1
go: github.com/libp2p/go-libp2p-peer@v2.4.0+incompatible: unknown revision v2.4.0
go: finding github.com/google/uuid v1.1.1
go: finding github.com/dgryski/go-farm v0.0.0-20190104051053-3adb47b1fb0f
go: finding github.com/ipfs/go-ipfs-exchange-offline v0.0.1
go: finding github.com/ipfs/go-blockservice v0.0.1
go: error loading module requirements
Makefile:11: recipe for target 'ipcs' failed
make: *** [ipcs] Error 1
```

@hinshun

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,0 +1,10 @@
+language: go
+
+go:
+- 1.12.x
+
+env:
+  - GO111MODULE=on
+
+script:
+  - make ipcs containerd-binary

--- a/go.mod
+++ b/go.mod
@@ -32,16 +32,10 @@ require (
 	github.com/ipfs/go-ipfs-http-client v0.0.1
 	github.com/ipfs/go-ipfs-util v0.0.1
 	github.com/ipfs/interface-go-ipfs-core v0.0.1
-	github.com/libp2p/go-flow-metrics v0.2.0 // indirect
-	github.com/libp2p/go-libp2p-crypto v2.0.1+incompatible // indirect
-	github.com/libp2p/go-libp2p-metrics v2.1.7+incompatible // indirect
-	github.com/libp2p/go-libp2p-peer v2.4.0+incompatible // indirect
-	github.com/libp2p/go-libp2p-protocol v1.0.0 // indirect
 	github.com/mistifyio/go-zfs v2.1.1+incompatible // indirect
 	github.com/moby/buildkit v0.3.3
 	github.com/multiformats/go-multiaddr v0.0.2 // indirect
-	github.com/multiformats/go-multiaddr-net v1.6.3 // indirect
-	github.com/multiformats/go-multihash v1.0.8
+	github.com/multiformats/go-multihash v0.0.1
 	github.com/opencontainers/go-digest v1.0.0-rc1
 	github.com/opencontainers/image-spec v1.0.1
 	github.com/opencontainers/runc v0.1.1 // indirect

